js/src/personal/riyuan/Riyuan.page.tsx

```diff
@@ -1,35 +1,15 @@
-import { Title, Text, MantineProvider } from '@mantine/core'
-import styles from './Riyuan.module.css'
+import { MantineProvider } from '@mantine/core'
 import { theme } from './theme.ts'
+import { Nav } from './src/navigation/Navigation.tsx'
+import { Home } from './src/home/home.tsx'
+import { About } from './src/about/about.tsx'
```

Comment:
see other comment about renaming riyuan/src as components and using absolute paths